test/PartnerConfiguration.spec.ts

```diff
@@ -18,6 +18,15 @@ const DEFAULT_DISCOUNT = 4;
 const DEFAULT_IS_UNICODE_SUPPORTED = false;
 const DEFAULT_FEE_PERCENTAGE = 5;
 
+const MIN_DURATION_CHANGED_EVENT = 'MinDurationChanged';
```

Comment:
nice. It could be in a separate file tho. in order to allow its usage outside this file.